Obleak.Fody/Extensions.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Mono.Cecil;
+using Mono.Cecil.Cil;
+using Mono.Cecil.Rocks;
+
+namespace Obleak.Fody
+{
+    internal static class Extensions
```

Comment:
Ya no dramas. Also one of the first extensions I write for every project I use which does LINQ. :tired_face: